packages/fetch/src/server.ts

```diff
@@ -6,14 +6,29 @@ if (!globalThis.fetch) {
         Object.assign(globalThis, { fetch });
         return fetch(...args);
       } catch (e) {
-        console.warn(
-          '"\x1b[33m⚠️ package missing to run createFetch on the server.\n Please run:\x1b[0m\n\nnpm i node-fetch\n"'
-        );
-        Object.assign(globalThis, { fetch: () => Promise.reject() });
+        Object.assign(globalThis, {
+          fetch: () => {
+            console.warn(
+              '"\x1b[33m⚠️ package missing to run createFetch on the server.\n Please run:\x1b[0m\n\nnpm i node-fetch\n"'
+            );
+            Promise.reject();
+          }
+        });
         return Promise.reject();
       }
     }
   });
 }
 
-export * from "./index";
+export { createFetch, FetchReturn, FetchOptions, RequestContext } from "./fetch";
+export {
+  withAbort,
+  withCatchAll,
+  withTimeout,
+  withRetry,
+  withRefetchEvent,
+  wrapFetcher,
+  wrapResource
+} from "./modifiers";
+export { withCache, withCacheStorage } from "./cache";
+export { fetchRequest, axiosRequest } from "./request";
```

Comment:
👀 

I advise you to set up this test suite to catch unmatched exports: https://github.com/solidjs-community/solid-primitives/blob/rework-visibility-observer/packages/intersection-observer/test/exports.test.ts

Reply:
That's great! It should be added to our template!